sfepy/discrete/problem.py

```diff
@@ -621,7 +621,8 @@ def update_equations(self, ts=None, ebcs=None, epbcs=None,
         graph_changed = self.equations.time_update(self.ts,
                                                    ebcs, epbcs, lcbcs,
                                                    functions, self,
-                                                   active_only=ac)
+                                                   active_only=ac,
+                                                   verbose=get_default_attr(self.conf, 'verbose', False))
```

Comment:
Do not change the default, please. You can use `self.conf.get('verbose', True)`. The same applies to `self.update_materials()` calls below.